frontend/tests/controllers-test/featuredChallengeCtrl.test.js

```
@@ -102,8 +103,7 @@ describe('Unit tests for featured challenge controller', function () {
             challengePhaseSuccess = null;
             challengePhaseSplitSuccess = null;
             vm = createController();
-            expect(utilities.storeData).toHaveBeenCalledWith('emailError', errorResponse.detail);
-            expect($state.go).toHaveBeenCalledWith('web.permission-denied');
+            expect(utilities.handlePermissionDeniedError).toHaveBeenCalled();
```

ESLint

[error] 'expect' is not defined.

(no-undef)